### PR TITLE
Shvdn3

### PR DESCRIPTION
**Pull request by GenesisEB**
This should get NativeUI ready for shvdn3

### CODE REVIEW

File: UIResText.cs [hunk 1]
```diff
@@ -23,7 +24,7 @@ public UIResText(string caption, Point position, float scale, Color color)
         }
 
         public UIResText(string caption, Point position, float scale, Color color, Font font, Alignment justify)
-            : base(caption, position, scale, color, font, false)
+            : base(caption, position, scale, color, font, 0)
```
Comment (GenesisEB) [inline UIResText.cs]: I don't know how the last variable changed, it would not accept a boolean with resharper so I just used 0.

File: UIMenu.cs [hunk 2]
```diff
@@ -1244,8 +1245,8 @@ public bool Visible
                 UpdateScaleform();
                 if (ParentMenu != null || !value) return;
                 if (!ResetCursorOnOpen) return;
-                Cursor.Position = new Point(Screen.PrimaryScreen.Bounds.Width/2, Screen.PrimaryScreen.Bounds.Height/2);
-                Function.Call(Hash._0x8DB8CFFD58B62552, 1);
+                Cursor.Position = new Point(System.Windows.Forms.Screen.PrimaryScreen.Bounds.Width/2, System.Windows.Forms.Screen.PrimaryScreen.Bounds.Height/2);
```
Comment (GenesisEB) [inline UIMenu.cs]: I don't know why this is using windows forms, doesn't Screen.Resolution work like the others?

Comment (Guad) [inline UIMenu.cs]: That's because I need the absolute mouse position in case the game is windowed.

**Comment by Guad** [comment]
Good job, thank you!

**Pull request merged**